text/1000-make-array-built-in.md

```diff
@@ -30,55 +27,42 @@ project-link: Leave as is
 suite: Leave as is
 -->
 
-# <RFC title>
+# Make `(array)` a built in helper 
 
 ## Summary
 
-> One paragraph explanation of the feature.
+Today, when using gjs/gts/`<template>`, in order to bind event listeners, folks _must import_ the `(array)` helper.
```

Comment:
Same here, do we really want `in order to bind event listeners`?

Reply:
Nope, copy paste mistake